Qt/mainwindow.cpp

```diff
@@ -293,28 +293,33 @@ void MainWindow::consoleAct()
 void MainWindow::raiseTopMost()
 {
 	setWindowState( (windowState() & ~Qt::WindowMinimized) | Qt::WindowActive);
-	raise();  
+	raise();
 	activateWindow();
 }
 
 void MainWindow::SetFullScreen(bool fullscreen) {
 	if (fullscreen) {
+#if !PPSSPP_PLATFORM(MAC)
 		menuBar()->hide();
```

Comment:
Qt uses native menubar on macOS by default, which is hidden automatically on fullscreen.